src/aws_utils/cloudwatch_utils.rs

```diff
@@ -9,6 +13,9 @@ pub const TX_REVERTED_METRIC: &str = "TransactionReverted";
 pub const TX_SUBMITTED_METRIC: &str = "TransactionSubmitted";
 pub const TX_STATUS_UNKNOWN_METRIC: &str = "TransactionStatusUnknown";
 pub const LATEST_BLOCK: &str = "LatestBlock";
+pub const EXECUTION_ATTEMPTED_METRIC: &str = "ExecutionAttempted";
+pub const EXECUTION_SKIPPED_ALREADY_FILLED_METRIC: &str = "ExecutionSkippedAlreadyFilled";
+pub const EXECUTION_SKIPPED_PAST_DEADLINE_METRIC: &str = "ExecutionSkippedPastDeadline";
```

Comment:
Can we add one for when we skip sending it to the executor if the route is not profitable at all?